test/arelastic/aggregations/aggregation_test.rb

```diff
@@ -19,4 +19,34 @@ def test_nested
 
     assert_equal expected, aggregation.as_elastic
   end
-end
\ No newline at end of file
+
+  def test_reverse_nested
+    aggregation = Arelastic::Aggregations::Min.new('smallest', 'field' => 'pets.weight').reverse_nested('pets')
+
+    expected = {
+      "smallest" => {
+        "reverse_nested" => {
+          "path" => "pets"
+        },
+        "aggs" => {
+          "smallest" => {
+            "min" => { "field" => "pets.weight" }
+          }
+        }
+      }
+    }
+
+    aggregation = Arelastic::Aggregations::Min.new('youngest', 'field' => 'age').reverse_nested
+
+    expected = {
+      "youngest" => {
+        "reverse_nested" => {},
+        "aggs" => {
+          "youngest" => {
+            "min" => { "field" => "age" }
+          }
+        }
+      }
+    }
+  end
+end
```

Comment:
min aggs don't support nested, so this should be changed to something more realistic.